JsonSchema.Generation/assembly.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("JsonSchema.Generation.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100d111b5991feb360746cf4535b9ec3196825996680b6599cf10748f90a58ec820431f81d21eff5229a3fd7c3e42e46a20584ac413ebe39b4c05905fd47b877bffed36bff23ad4ad14aad3deed8f24ed40a1371daad79a41172b2cdf0db4b664198f38f3a19419457131f8ee4558e628b5c986bb574d4e151b650eae6aa1f5ccbf")]
```

Comment:
Why?

Reply:
removed, now that there is no internal class anymore